frontend/src/app/oe/interfaces/op-energy.interface.ts

```diff
@@ -50,3 +50,8 @@ export interface NavigationObject {
     },
   },
 }
+
+export interface BlockHeights {
```

Comment:
BlockHeights is wrong name here: BlockHeights means list of BlockHeight. It seems, that it should be BlockSpan instead, as in the backend